encoders/image/ImageTorchEncoder/README.md

```diff
@@ -1,3 +1,56 @@
 # ImageTorchEncoder
 
-`ImageTorchEncoder` encodes data from a ndarray, potentially B x (Channel x Height x Width) into a 
\ No newline at end of file
+`ImageTorchEncoder` encodes data from a ndarray, potentially BatchSize x (Channel x Height x Width) into a ndarray of `BatchSize * d`. Internally, :class:`ImageTorchEncoder` wraps the models from `tensorflow.keras.applications`. https://keras.io/applications/
+
+
+## Usage:
+
+Initialise this Executor specifying parameters i.e.:
+
+| `param_name`  | `param_remarks` |
+| ------------- | ------------- |
+| `model_path`  | the directory path of the model in the `SavedModel` format  |
+| `model_name`  | includes `resnet18`, `alexnet`, `squeezenet1_0`, `vgg16`, `densenet161`, `inception_v3` |
+| `channel_axis`| axis of the color channel  |
+| `pool_strategy` | strategy of pooling operation |
+
+The model is pre-trained on [MobileNetV2] data
```

Comment:
missing link for `MobileNetV2`

Reply:
MobileNetV2 data makes no sense, the dataset is ImageNet. But anyway I think is not worth, I would say `model` is one of `resnet18`, `alexnet`, `squeezenet1_0`, `vgg16`, `densenet161`, `inception_v3`

Reply:
I've updated the doc to reflect the right models.